docs/extensions/server/events.rst

```diff
@@ -55,8 +55,8 @@ To create custom EventHandlerProvider class derived from the above interface, we
 
    <dependency>
       <groupId>org.apache.kyuubi</groupId>
-      <artifactId>kyuubi-event_2.12</artifactId>
-      <version>1.7.0-incubating</version>
+      <artifactId>kyuubi-events_2.12</artifactId>
+      <version>1.7.0</version>
```

Comment:
You can use the directive - ` release`.

Reply:
Tried to use the directive， but it is not working (in code block).

```
<version>| release |</version>

<version>|release|</version>

<version>\ | release \ |</version>
```

Reply:
It works.

Reply:
OK, addressed. Since the directive requries `parsed-literal`, replace the `code-block` to `parsed-literal`